rfc.tex

```diff
@@ -25,11 +25,12 @@ \section{Decision Making Process}
 The DM Systems Engineer is tasked with looking at RFCs on a weekly basis to ensure that proposed RFCs are not languishing and that implemented RFCs are correctly marked as such.
 The \emph{Implemented} state is recognized by scanning all the \emph{Adopted} RFCs and seeing that all triggered work has been completed.
 
-The RFC process has been extremely successful with N RFCs filed since 201? and M implemented.
-We have found that some RFCs are filed asking for changes that the proposer feels are a good change to make but which they themselves are not going to be repsonsible for implementing.
+The RFC process has been extremely successful with 467 RFCs filed since 2014, with 321 implemented and 44 withdrawn.
```

Comment:
Begs the question of what is going on with the other 102...

Reply:
89 adopted, 5 flagged. 7 retired (which is not quite the same as Withdrawn but may as well be) 1 is proposed.